packages/dialog/src/FzConfirmDialog.vue

```diff
@@ -65,7 +65,8 @@ const props = withDefaults(defineProps<FzConfirmDialogProps>(), {
   disableConfirm: false,
   confirmButtonEnabled: true,
   doesCancelButtonCloseDialog: true,
-  doesConfirmButtonCloseDialog: true
+  doesConfirmButtonCloseDialog: true,
+  unrenderOnClose: true
```

Comment:
non è un cambio potenzialmente distruttivo? molti dialog che abbiamo sostituito li abbiamo corretti per farli funzionare con una dialog sempre presente nel DOM. Per sicurezza di default manterrei questa opzione disattivata

Reply:
Avevo pensato così anch'io ma aveva suggerito così @Trebh.
Se non mettiamo il default a true, dobbiamo andare a modificare ogni singola modale.
E non penso che in ogni caso ci credi problemi sulle modali esistenti.
Farei un giro di smoke tests e se va tutto bene rilascerei così, almeno così eravamo rimasti nell'ultimo meeting.